### PR TITLE
Bug 1196569 - Private Browsing product page

### PR DESCRIPTION
**Pull request by malena**
https://bugzilla.mozilla.org/show_bug.cgi?id=1196569

### CODE REVIEW

File: bedrock/firefox/templates/firefox/private-browsing.html [hunk 1]
```diff
@@ -0,0 +1,136 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% from "macros.html" import fxfamilynav with context %}
+
+{% extends "/firefox/base-resp.html" %}
+
+{% block page_title %}{{ _('Keep your Firefox in sync') }}{% endblock %}
```
Comment (flodolo) [inline bedrock/firefox/templates/firefox/private-browsing.html]: Is this correct?

File: bedrock/firefox/templates/firefox/private-browsing.html [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
+  <section class="intro">
+    <div class="container">
+      <div class="header">
+        <h1>
+          {{ _('More protection.') }}
+          <br>
+          {{ _('The most privacy.') }}
+          <br>
+          {{ _('Only from Firefox.') }}
```
Comment (flodolo) [inline bedrock/firefox/templates/firefox/private-browsing.html]: We have already [exposed](More protection. <br>The most privacy. <br>Only from Firefox.) a similar string for the campaign landing page, I'd suggest to reuse the same
`More protection. <br>The most privacy. <br>Only from Firefox.`

File: bedrock/firefox/templates/firefox/private-browsing.html (hunk 1 above)
Comment (craigcook) [inline bedrock/firefox/templates/firefox/private-browsing.html]: No, I missed that (copied from the sync page). I'm not sure what the title of this page should be...

**Comment by malena** [comment]
This PR has been replaced with https://github.com/mozilla/bedrock/pull/3391

**Pull request closed without merging**